<?xml version="1.0" encoding="us-ascii"?>
<html><head><title>Generating a Haploview</title>
<meta content="text/html; charset=us-ascii" http-equiv="Content-Type" /><link rel="stylesheet" type="text/css" href="../Theme/nav.css" /><link rel="stylesheet" type="text/css" href="../Theme/popupmenu.css" /><link rel="stylesheet" type="text/css" href="../Theme/lf.css" /><link rel="stylesheet" type="text/css" href="../LinksExt/d2hTarget.css" /><link rel="stylesheet" type="text/css" href="../LinksExt/OpenSource_tranSMART_User_Guide.css" />
<script type="text/javascript">
//<![CDATA[



var _needIndentation = false;




//]]></script>

<script language="JavaScript" src="../linker.js" type="text/JavaScript"></script>

<script language="JavaScript" src="../special.js" type="text/JavaScript"></script>

<script language="JavaScript" src="../common.js" type="text/JavaScript"></script>

<meta name="Generator" content="ComponentOne Doc-To-Help v.2011.1.0.624" />
<meta name="D2HTheme" content="(Default) v.150" />
</head>
<body style="PADDING-BOTTOM: 0px; BACKGROUND-COLOR: #535353; MARGIN: 7px; PADDING-LEFT: 0px; PADDING-RIGHT: 0px; OVERFLOW: hidden; PADDING-TOP: 0px" onload="d2hOnLoadToolbar(event);" relpart="../">
<script language="javascript" type="text/javascript">
//<![CDATA[



d2hTopicPreOpen();



d2hTrySyncTOC();







var prev = true;





var next = true;



if (typeof d2hInitMainThemeHandlers != "undefined")

    d2hInitMainThemeHandlers(prev, next);

d2hAdjustTopicBorders(document);    



if (typeof d2hLoadWindows != "undefined")

    d2hLoadWindows();

if (typeof d2hLoadNavUrls != "undefined")

    d2hLoadNavUrls();




//]]></script>

<div style="PADDING-BOTTOM: 0px; BACKGROUND-COLOR: white; MARGIN: 0px; PADDING-LEFT: 0px; WIDTH: 100%; PADDING-RIGHT: 0px; HEIGHT: 100%; OVERFLOW: auto; PADDING-TOP: 0px" id="D2HTopicOuterBody" class="d2h_outer_topic">
<div id="nsbanner" class="d2hNavigatorTopic">
<div style="PADDING-BOTTOM: 0px; PADDING-LEFT: 0px; PADDING-RIGHT: 0px; PADDING-TOP: 0px" id="TitleRow" class="d2h_main_titlerow">
<div style="DISPLAY: none"><a id="D2HPreviousAnchor" href="hazardratioandrelativerisk.htm"></a><a id="D2HNextAnchor" href="runningthesnpviewer.htm"></a></div></div>
<script language="javascript" type="text/javascript">
//<![CDATA[

if (typeof d2hProcessTopicNavForCSH != "undefined")

    d2hProcessTopicNavForCSH();


//]]></script>
</div>
<div style="PADDING-BOTTOM: 10px; PADDING-LEFT: 22px; PADDING-RIGHT: 10px; HEIGHT: 100%; PADDING-TOP: 5px" id="nstext" class="d2h_main_nstext" valign="bottom">
<div id="d2h_breadcrumbs" class="d2h_breadcrumbs"><span class="d2h_breadcrumbsLabel"><b><span style="COLOR: #000080">User's 
Guide</span></b>&#160;&#160;</span><a class="d2h_breadcrumbsNormal" href="chapter3datasetexplorer.htm">Chapter 3: Dataset Explorer</a></div>
<h2><a name="_Toc314210661" id="_Toc314210661"></a><a name="_Toc297057917" id="_Toc297057917">Generating a 
Haploview</a></h2>
<table style="BORDER-BOTTOM: medium none; BORDER-LEFT: medium none; BORDER-COLLAPSE: collapse; BORDER-TOP: medium none; BORDER-RIGHT: medium none" class="MsoTableGrid" border="0" cellspacing="0" cellpadding="0">
  <tbody>
  <tr>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 55.8pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="74">
      <p style="PAGE-BREAK-AFTER: avoid; TEXT-ALIGN: center" class="MsoNormal" align="center"><img border="0" src="../ImagesExt/image17_84.png" alt="" /></p></td>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 387pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="516">
      <p style="PAGE-BREAK-AFTER: avoid" class="MsoNormal">The GPL version of the 
      tranSMART open source software does not include the Broad Institute&#8217;s 
      GenePattern software.&#160; GenePattern is needed to use some of 
      tranSMART&#8217;s advanced scientific workflows (Heat Map Viewer, SNP Viewer, 
      and Integrated Genomics Viewer). To use these features in tranSMART, 
      download the GenePattern software from The Broad Institute&#8217;s web site 
      (http://www.broadinstitute.org).</p></td></tr></tbody></table>
<p class="MsoNormal">A haploview allows you to analyze the differences in allele 
frequency in two or more loci from one sample to the next. Haploviews are 
generated for SNP data. </p>
<p class="MsoNormal">You may run multiple visualizations in the background while 
continuing to use Dataset Explorer. For more information on running 
visualizations in the background, see <span class="xRef"><a href="asynchronousoperations.htm"><span class="xRef">Asynchronous 
Operations</span></a></span>.</p>
<p style="tab-stops: 0pt" class="ListNumStart">To generate a haploview:</p>
<p style="tab-stops: 18.0pt 36.0pt 54.0pt 72.0pt 90.0pt 108.0pt" class="MsoListNumber">1.&#160; Run tranSMART, then click the <span class="Bold">Dataset Explorer</span> tab.</p>
<p style="tab-stops: 18.0pt 36.0pt 54.0pt 72.0pt 90.0pt 108.0pt" class="MsoListNumber">2.&#160; Select the study of interest.</p>
<p style="tab-stops: 18.0pt 36.0pt 54.0pt 72.0pt 90.0pt 108.0pt" class="MsoListNumber">3.&#160; Define the cohorts whose data points will be 
represented in the haploview.</p>
<p style="PAGE-BREAK-AFTER: avoid; tab-stops: 18.0pt 36.0pt 54.0pt 72.0pt 90.0pt 108.0pt" class="MsoListNumber">4.&#160; Do one of the following:</p>
<p style="PAGE-BREAK-AFTER: avoid; MARGIN-BOTTOM: 6pt" class="MsoNormalIndent"><span class="Bold">To base the haploview on one or more 
genes:</span></p>
<p style="PAGE-BREAK-AFTER: avoid; MARGIN-BOTTOM: 6pt" class="MsoListNumber2">a.&#160; Click <span class="Bold">Advanced &gt; 
Haploview</span>.</p>
<p style="MARGIN-BOTTOM: 6pt" class="MsoListNumber2">b.&#160; Select one or more 
genes on which to base the haploview &#8211; for example:</p>
<p class="NormalIndent2"><img border="0" src="../ImagesExt/image17_129.jpg" alt="" /></p>
<p class="MsoListNumber2">c.&#160; Click <span class="Bold">OK</span>.</p>
<table style="BORDER-BOTTOM: medium none; BORDER-LEFT: medium none; BORDER-COLLAPSE: collapse; BORDER-TOP: medium none; BORDER-RIGHT: medium none" class="MsoTableGrid" border="0" cellspacing="0" cellpadding="0">
  <tbody>
  <tr>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 54.9pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="73">
      <p style="PAGE-BREAK-AFTER: avoid; TEXT-ALIGN: center" class="MsoNormal" align="center"><img border="0" alt="Description: C:\Users\bkingsbury\Dropbox\Stuff\Recombinant\note.png" src="../ImagesExt/image17_0.jpg" /></p></td>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 387.9pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="517">
      <p style="PAGE-BREAK-AFTER: avoid" class="MsoNormal"><span style="COLOR: black">If you see a message like the following, a haploview 
      cannot be generated for the gene(s) you 
selected</span>:</p></td></tr></tbody></table>
<p style="PAGE-BREAK-AFTER: avoid; MARGIN: 9pt -36pt 18pt 54pt" class="MsoNormalIndent">&#160;&#160;&#160;&#160; <img border="0" src="../ImagesExt/image17_130.jpg" alt="" /></p>
<p style="PAGE-BREAK-AFTER: avoid" class="MsoNormalIndent"><span class="Bold">To 
base the haploview on SNPs or nucleotides:</span></p>
<p style="PAGE-BREAK-AFTER: avoid" class="MsoListNumber2"><span class="Bold"><span style="FONT-WEIGHT: normal">d.&#160; Select a </span>SNP ID</span><span class="Bold"><span style="FONT-WEIGHT: normal"> or </span>nucleotide </span><span class="Bold"><span style="FONT-WEIGHT: normal">and drag it into a subset 
box</span>:</span><span class="Bold"><span style="FONT-WEIGHT: normal"></span></span></p>
<p class="NormalIndent2"><img border="0" src="../ImagesExt/image17_131.jpg" alt="" /><span class="Bold"><span style="FONT-WEIGHT: normal"></span></span></p>
<p class="MsoListNumber2"><span class="Bold"><span style="FONT-WEIGHT: normal">e.&#160; Click</span> Advanced &gt; 
Haploview.</span><span class="Bold"><span style="FONT-WEIGHT: normal"></span></span></p>
<table style="BORDER-BOTTOM: medium none; BORDER-LEFT: medium none; BORDER-COLLAPSE: collapse; BORDER-TOP: medium none; BORDER-RIGHT: medium none" class="MsoTableGrid" border="0" cellspacing="0" cellpadding="0">
  <tbody>
  <tr>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 54.9pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="73">
      <p style="TEXT-ALIGN: center" class="MsoNormal" align="center"><img border="0" alt="Description: C:\Users\bkingsbury\Dropbox\Stuff\Recombinant\note.png" src="../ImagesExt/image17_0.jpg" /></p></td>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 387.9pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="517">
      <p class="MsoNormal"><span style="COLOR: black">Not all trials support the 
      selection of individual SNPs and 
nucleotides.</span></p></td></tr></tbody></table>
<p style="PAGE-BREAK-AFTER: avoid" class="MsoNormal">For a more detailed 
description of what a haploview represents, click the Information icon 
(&#160;<img border="0" src="../ImagesExt/image17_132.png" alt="" />&#160;) in the 
haploview:</p>
<p class="MsoNormal"><img border="0" src="../ImagesExt/image17_133.jpg" alt="" /></p></div></div></body></html>
