<?xml version="1.0" encoding="us-ascii"?>
<html><head><title>Heat Maps Based on Gene Expression Data</title>
<meta content="text/html; charset=us-ascii" http-equiv="Content-Type" /><link rel="stylesheet" type="text/css" href="../Theme/nav.css" /><link rel="stylesheet" type="text/css" href="../Theme/popupmenu.css" /><link rel="stylesheet" type="text/css" href="../Theme/lf.css" /><link rel="stylesheet" type="text/css" href="../LinksExt/d2hTarget.css" /><link rel="stylesheet" type="text/css" href="../LinksExt/OpenSource_tranSMART_User_Guide.css" />
<script type="text/javascript">
//<![CDATA[



var _needIndentation = false;




//]]></script>

<script language="JavaScript" src="../linker.js" type="text/JavaScript"></script>

<script language="JavaScript" src="../special.js" type="text/JavaScript"></script>

<script language="JavaScript" src="../common.js" type="text/JavaScript"></script>

<meta name="Generator" content="ComponentOne Doc-To-Help v.2011.1.0.624" />
<meta name="D2HTheme" content="(Default) v.150" />
</head>
<body style="PADDING-BOTTOM: 0px; BACKGROUND-COLOR: #535353; MARGIN: 7px; PADDING-LEFT: 0px; PADDING-RIGHT: 0px; OVERFLOW: hidden; PADDING-TOP: 0px" onload="d2hOnLoadToolbar(event);" relpart="../">
<script language="javascript" type="text/javascript">
//<![CDATA[



d2hTopicPreOpen();



d2hTrySyncTOC();







var prev = true;





var next = true;



if (typeof d2hInitMainThemeHandlers != "undefined")

    d2hInitMainThemeHandlers(prev, next);

d2hAdjustTopicBorders(document);    



if (typeof d2hLoadWindows != "undefined")

    d2hLoadWindows();

if (typeof d2hLoadNavUrls != "undefined")

    d2hLoadNavUrls();




//]]></script>

<div style="PADDING-BOTTOM: 0px; BACKGROUND-COLOR: white; MARGIN: 0px; PADDING-LEFT: 0px; WIDTH: 100%; PADDING-RIGHT: 0px; HEIGHT: 100%; OVERFLOW: auto; PADDING-TOP: 0px" id="D2HTopicOuterBody" class="d2h_outer_topic">
<div id="nsbanner" class="d2hNavigatorTopic">
<div style="PADDING-BOTTOM: 0px; PADDING-LEFT: 0px; PADDING-RIGHT: 0px; PADDING-TOP: 0px" id="TitleRow" class="d2h_main_titlerow">
<div style="DISPLAY: none"><a id="D2HPreviousAnchor" href="heatmapsbasedonrbmdata.htm"></a><a id="D2HNextAnchor" href="heatmapsbasedonproteomicsdata.htm"></a></div></div>
<script language="javascript" type="text/javascript">
//<![CDATA[

if (typeof d2hProcessTopicNavForCSH != "undefined")

    d2hProcessTopicNavForCSH();


//]]></script>
</div>
<div style="PADDING-BOTTOM: 10px; PADDING-LEFT: 22px; PADDING-RIGHT: 10px; HEIGHT: 100%; PADDING-TOP: 5px" id="nstext" class="d2h_main_nstext" valign="bottom">
<div id="d2h_breadcrumbs" class="d2h_breadcrumbs"><span class="d2h_breadcrumbsLabel"><b><span style="COLOR: #000080">User's 
Guide</span></b>&#160;&#160;</span><a class="d2h_breadcrumbsNormal" href="chapter3datasetexplorer.htm">Chapter 3: Dataset Explorer</a><span class="d2h_breadcrumbsLinkSeparator"> &gt; </span><a class="d2h_breadcrumbsNormal" href="generatingheatmaps.htm">Generating Heat Maps</a><span class="d2h_breadcrumbsLinkSeparator"> &gt; </span><a class="d2h_breadcrumbsNormal" href="instructionsforgeneratingheatmaps.htm">Instructions for Generating Heat 
Maps</a></div>
<h4>Heat Maps Based on Gene Expression Data</h4>
<p style="PAGE-BREAK-AFTER: avoid" class="MsoNormal">You generate a heat map based 
on gene expression data from a source such as Affymetrix, Illumina, or 
Agilent.</p>
<p style="tab-stops: 0pt" class="ListNumStart">To generate a heat map based on 
gene expression data:</p>
<p style="tab-stops: 18.0pt 36.0pt 54.0pt 72.0pt 90.0pt 108.0pt" class="MsoListNumber">1.&#160; Define one or both subsets, as described earlier in 
this chapter.</p>
<table style="BORDER-BOTTOM: medium none; BORDER-LEFT: medium none; BORDER-COLLAPSE: collapse; BORDER-TOP: medium none; BORDER-RIGHT: medium none" class="MsoTableGrid" border="0" cellspacing="0" cellpadding="0">
  <tbody>
  <tr>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 54.9pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="73">
      <p style="TEXT-ALIGN: center" class="MsoNormal" align="center"><img border="0" alt="Description: C:\Users\bkingsbury\Dropbox\Stuff\Recombinant\note.png" src="../ImagesExt/image17_0.jpg" /></p></td>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 387.9pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="517">
      <p class="MsoNormal"><span style="COLOR: black">If you intend to generate a 
      Comparative Marker Selection heat </span>map, you must define both 
      subsets.</p></td></tr></tbody></table>
<p style="tab-stops: 18.0pt 36.0pt 54.0pt 72.0pt 90.0pt 108.0pt" class="MsoListNumber">2.&#160; In the navigation tree, select the sample(s) and/or 
timepoint(s) for the gene expression platform and drag them into the subset 
definition boxes.</p>
<table style="BORDER-BOTTOM: medium none; BORDER-LEFT: medium none; BORDER-COLLAPSE: collapse; BORDER-TOP: medium none; BORDER-RIGHT: medium none" class="MsoTableGrid" border="0" cellspacing="0" cellpadding="0">
  <tbody>
  <tr>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 54.9pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="73">
      <p style="TEXT-ALIGN: center" class="MsoNormal" align="center"><img border="0" alt="Description: C:\Users\bkingsbury\Dropbox\Stuff\Recombinant\note.png" src="../ImagesExt/image17_0.jpg" /></p></td>
    <td style="PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 387.9pt; PADDING-RIGHT: 5.4pt; PADDING-TOP: 0pt" valign="top" width="517">
      <p class="MsoNormal"><span style="COLOR: black">Alternatively, you can omit 
      some or all of the gene expression samples/ timepoints </span>from the 
      subset definitions, and instead define them in a dialog after you select 
      the type of heat map to generate.</p></td></tr></tbody></table>
<p style="PAGE-BREAK-AFTER: avoid" class="MsoNormalIndent">To select gene 
expression samples and/or timepoints from the navigation tree and add them to 
your subset definitions:</p>
<p class="MsoListNumber2">a.&#160; Open the appropriate platform node (such as 
<span class="Bold">Affymetrix...,</span> <span class="Bold">Illumina...,</span> or 
<span class="Bold">Agilent...</span>) under the <span class="Bold">Biomarker 
Data</span> branch of the navigation tree for the study of interest.</p>
<p class="MsoListNumber2">b.&#160; Drag a node containing a sample type or 
timepoint into Subset 1 &#8211; for example:</p>
<p class="NormalIndent2">Nodes containing sample types:</p>
<p class="NormalIndent2"><img border="0" src="../ImagesExt/image17_103.png" alt="" /></p>
<p class="NormalIndent2">Nodes containing timepoints:</p>
<p class="NormalIndent2"><img border="0" src="../ImagesExt/image17_104.png" alt="" /></p>
<p style="PAGE-BREAK-AFTER: avoid; tab-stops: 18.0pt 36.0pt 54.0pt 72.0pt 90.0pt 108.0pt" class="MsoListNumber">3.&#160; Optionally, repeat the previous step to include 
additional sample types/timepoints in the heat map for Subset 2.</p>
<p style="PAGE-BREAK-AFTER: avoid; tab-stops: 18.0pt 36.0pt 54.0pt 72.0pt 90.0pt 108.0pt" class="MsoListNumber">4.&#160; Click the Dataset Explorer <span class="Bold">Advanced</span> button, then select the type of heat map you want to 
generate:</p>
<p class="MsoNormalIndent"><img border="0" src="../ImagesExt/image17_105.jpg" alt="" /></p>
<p style="PAGE-BREAK-AFTER: avoid" class="MsoNormalIndent">One of the following 
responses occurs, depending on the type of heat map you selected:</p>
<table style="BORDER-BOTTOM: medium none; BORDER-LEFT: medium none; BORDER-COLLAPSE: collapse; MARGIN-LEFT: 23.4pt; BORDER-TOP: medium none; BORDER-RIGHT: medium none" class="tranSMARTTable" border="1" cellspacing="0" cellpadding="0">
  <thead>
  <tr>
    <td style="BORDER-BOTTOM: windowtext 2.25pt double; BORDER-LEFT: windowtext 1.5pt solid; PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 114pt; PADDING-RIGHT: 5.4pt; BORDER-TOP: windowtext 1.5pt solid; BORDER-RIGHT: black 1pt solid; PADDING-TOP: 0pt" width="152">
      <p class="TableHeading">Menu Choice<span style="COLOR: windowtext; FONT-SIZE: 10pt"></span></p></td>
    <td style="BORDER-BOTTOM: windowtext 2.25pt double; BORDER-LEFT: medium none; PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 300pt; PADDING-RIGHT: 5.4pt; BORDER-TOP: windowtext 1.5pt solid; BORDER-RIGHT: windowtext 1.5pt solid; PADDING-TOP: 0pt" width="400">
      <p class="TableHeading">&#160;Response<span style="COLOR: windowtext; FONT-SIZE: 10pt"></span></p></td></tr></thead>
  <tbody>
  <tr style="PAGE-BREAK-INSIDE: avoid">
    <td style="BORDER-BOTTOM: black 1pt solid; BORDER-LEFT: windowtext 1.5pt solid; PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 114pt; PADDING-RIGHT: 5.4pt; BORDER-TOP: medium none; BORDER-RIGHT: black 1pt solid; PADDING-TOP: 0pt" valign="top" width="152">
      <p class="TableText">Heatmap (standard) or Hierarchical Clustering<span style="FONT-SIZE: 10pt"></span></p></td>
    <td style="BORDER-BOTTOM: black 1pt solid; BORDER-LEFT: medium none; PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 300pt; PADDING-RIGHT: 5.4pt; BORDER-TOP: medium none; BORDER-RIGHT: windowtext 1.5pt solid; PADDING-TOP: 0pt" valign="top" width="400">
      <p class="TableText">The Compare Subsets-Pathway Selection dialog 
      appears:<span style="FONT-SIZE: 10pt"></span></p>
      <p class="TableText"><span style="FONT-SIZE: 10pt"><img border="0" src="../ImagesExt/image17_106.jpg" alt="" /></span><span style="FONT-SIZE: 10pt"></span></p>
      <p style="MARGIN-TOP: 12pt" class="TableText"><b><span style="COLOR: #1f497d">Actions you take:</span></b></p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>Select the <span class="Bold">MRNA</span> platform and other 
      parameters for the heat map, or accept the defaults, if any.&#160; </p>
      <p class="TableIndent">Multiple GPL platforms, samples, tissue types, and 
      timepoints can be selected within a field.&#160; (Click to select, click 
      again to deselect.)</p>
      <p class="TableIndent">If you leave<span class="Bold"> one of the above fields 
      </span>blank, all possible values for the field will be factored into the 
      heat map.</p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>To filter the dataset by a gene or pathway, type part or all of a 
      gene or pathway name in the <span class="Bold">Select a</span> <span class="Bold">Gene/Pathway</span> field, then select the full name from the 
      dropdown:<span style="FONT-SIZE: 10pt"></span></p>
      <p class="TableIndent"><img border="0" src="../ImagesExt/image17_107.jpg" alt="" /><span style="FONT-SIZE: 10pt"></span></p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>Click <span class="Bold">OK</span>.&#160; After a few seconds, the 
      heat map appears.<span style="FONT-SIZE: 10pt"></span></p></td></tr>
  <tr style="PAGE-BREAK-INSIDE: avoid">
    <td style="BORDER-BOTTOM: black 1pt solid; BORDER-LEFT: windowtext 1.5pt solid; PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 114pt; PADDING-RIGHT: 5.4pt; BORDER-TOP: medium none; BORDER-RIGHT: black 1pt solid; PADDING-TOP: 0pt" valign="top" width="152">
      <p style="PAGE-BREAK-AFTER: avoid" class="TableText">K-Means Clustering<span style="FONT-SIZE: 10pt"></span></p></td>
    <td style="BORDER-BOTTOM: black 1pt solid; BORDER-LEFT: medium none; PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 300pt; PADDING-RIGHT: 5.4pt; BORDER-TOP: medium none; BORDER-RIGHT: windowtext 1.5pt solid; PADDING-TOP: 0pt" valign="top" width="400">
      <p class="TableText">The Compare Subsets-Pathway Selection dialog 
      appears:</p>
      <p style="PAGE-BREAK-AFTER: avoid" class="TableText"><img border="0" src="../ImagesExt/image17_108.jpg" alt="" />&#160;<span style="FONT-SIZE: 10pt"></span></p>
      <p style="PAGE-BREAK-AFTER: avoid; MARGIN-TOP: 12pt" class="TableText"><b><span style="COLOR: #1f497d">Actions you 
      take:</span></b></p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>Select the <span class="Bold">MRNA</span> platform and other 
      parameters for the heat map, or accept the defaults, if any.&#160; </p>
      <p class="TableIndent">Multiple GPL platforms, samples, tissue types, and 
      timepoints can be selected within a field.&#160; (Click to select, click 
      again to deselect.)</p>
      <p class="TableIndent">If you leave<span class="Bold"> one of the above fields 
      </span>blank, all possible values for the field will be factored into the 
      heat map.</p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>To filter the dataset by a gene or pathway, type part or all of a 
      gene or pathway name in the <span class="Bold">Select a</span> <span class="Bold">Gene/Pathway</span> field, then select the full name from the 
      dropdown.<span style="FONT-SIZE: 10pt"></span></p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>Use the <span class="Bold">Select the number of Clusters</span> field 
      to specify the number of clusters to view in the heat map.<span style="FONT-SIZE: 10pt"></span></p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>Click <span class="Bold">Run Workflow</span>.&#160; After a few 
      seconds, the heat map appears.<span style="FONT-SIZE: 10pt"></span></p></td></tr>
  <tr style="PAGE-BREAK-INSIDE: avoid">
    <td style="BORDER-BOTTOM: windowtext 1.5pt solid; BORDER-LEFT: windowtext 1.5pt solid; PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 114pt; PADDING-RIGHT: 5.4pt; BORDER-TOP: medium none; BORDER-RIGHT: black 1pt solid; PADDING-TOP: 0pt" valign="top" width="152">
      <p class="TableText">Comparative Marker Selection<span style="FONT-SIZE: 10pt"></span></p></td>
    <td style="BORDER-BOTTOM: windowtext 1.5pt solid; BORDER-LEFT: medium none; PADDING-BOTTOM: 0pt; PADDING-LEFT: 5.4pt; WIDTH: 300pt; PADDING-RIGHT: 5.4pt; BORDER-TOP: medium none; BORDER-RIGHT: windowtext 1.5pt solid; PADDING-TOP: 0pt" valign="top" width="400">
      <p class="TableText">The Compare Subsets-Pathway Selection dialog 
      appears:</p>
      <p class="TableText"><img border="0" src="../ImagesExt/image17_109.jpg" alt="" /></p>
      <p style="PAGE-BREAK-AFTER: avoid; MARGIN-TOP: 12pt" class="TableText"><b><span style="COLOR: #1f497d">Actions you 
      take:</span></b></p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>Select the <span class="Bold">MRNA</span> platform and other 
      parameters for the heat map, or accept the defaults, if any.&#160; </p>
      <p class="TableIndent">Multiple GPL platforms, samples, tissue types, and 
      timepoints can be selected within a field.&#160; (Click to select, click 
      again to deselect.)</p>
      <p class="TableIndent">If you leave<span class="Bold"> one of the above fields 
      </span>blank, all possible values for the field will be factored into the 
      heat map.</p>
      <p style="tab-stops: 12.0pt" class="TableBullet"><span style="POSITION: relative; FONT-FAMILY: 'Arial','sans-serif'; FONT-SIZE: 8pt; TOP: -2pt">&#9632;&#160;&#160; 
      </span>Click <span class="Bold">Run Workflow</span>.&#160; A heat map and a 
      Comparative Marker Selection viewer are displayed. </p>
      <p class="TableText"><span class="Bold"><span style="COLOR: #1f497d">Note:</span></span>&#160; Due to the large amount 
      of data being processed, it may take several minutes for the 
      visualizations to appear.<span style="FONT-SIZE: 10pt"></span></p></td></tr></tbody></table>
<p class="Spacer">&#160;</p></div></div></body></html>
